### ISSUE TITLE
AWS Billing module `EstimatedCharges` field doesn't match integration

### ISSUE THREAD

**Issue by legoguy1000** [elastic/integrations#1592]
The Metricbeat AWS Billing module sets the field `EstimatedCharges` as a Long.  The AWS Billing Integration data stream for the agent sets `EstimatedCharges.max` as a Long.  User reported an error with the AWS agent integration https://elasticstack.slack.com/archives/CNTP6NLTZ/p1631079578142800?thread_ts=1631079307.142600&cid=CNTP6NLTZ.
```
(status=400): {"type":"mapper_parsing_exception","reason":"object mapping for [aws.billing.EstimatedCharges] tried to parse field [EstimatedCharges] as object, but found a concrete value"}
```
The change in the Metricbeat module from `EstimatedCharges.max` to `EstimatedCharges` was made in https://github.com/elastic/beats/pull/20527 
The AWS Integration needs to update its field mapping to match the Metricbeat module.

**Comment by legoguy1000** [comment]
@kaiyan-sheng thoughts?

**Comment by kaiyan-sheng** [comment]
Yes this is a bug caused by not syncing the change between beats and the integration 👍 Thanks for catching it! Will fix it right away.

**Comment by legoguy1000** [comment]
I have a branch already, just hadn't pushed it yet. I can push it shortly for u too look at if u want or if there are other changes u want to make as well, I can just toss it.

**Comment by kaiyan-sheng** [comment]
@legoguy1000 😂 sorry!!! I just saw your comment...!!

**Comment by legoguy1000** [comment]
> @legoguy1000 😂 sorry!!! I just saw your comment...!!

no problem, i figured there were other changes that needed to be made too.  Glad it was fixed.

**Comment by kaiyan-sheng** [comment]
Thank you for creating the issue!!!

**Comment by threenub** [comment]
Thanks all - as the user that reported it let me know if I can help or test in any way.

**Comment by threenub** [comment]
Hi @kaiyan-sheng -- how long roughly until this fix makes it into the repo?

**Comment by kaiyan-sheng** [comment]
@threenub The PR is merged into `0.10.6` version of the AWS package, which is still in staging right now. It will be pushed into production in next day or two.
